### PR TITLE
chore(commons): migrate hugegraph-commons

### PR DESCRIPTION
**Pull request by VGalaxies**
**TODO:**
After this PR merged, change the README in commons & pin a [issue](https://github.com/hugegraph/hugegraph-hubble/issues/371) to notify all users/devs to know the context (also update the repo description...)

Finally, we may need start a discussion for marking commons repo in achieved status

Refer:
- https://github.com/apache/incubator-hugegraph-commons
- https://github.com/hugegraph/hugegraph-hubble 
image

### CODE REVIEW

**Comment by javeme** [comment]
Is hugegraph-commons still releasing versions frequently? we need to merge it into hugegraph only if yes.

File: .asf.yaml
Comment (javeme) [inline .asf.yaml:45]: prefer to rename to build-server and build-commons in ci.yml

Comment (imbajin) [inline .asf.yaml:45]: maybe we need ensure it in fork repo to prevent `dead-lock` (after merge it)

File: .github/workflows/commons-ci.yml
Comment (imbajin) [inline .github/workflows/commons-ci.yml:43]: we need visit stage here?

Comment (VGalaxies) [reply .github/workflows/commons-ci.yml:43]: commons-rpc depends on commons-commons, which currently only exists in the maven stage repository

File: .github/workflows/commons-ci.yml
Comment (javeme) [inline .github/workflows/commons-ci.yml:1]: prefer "HugeGraph-Commons CI", and please also improve the names in other ci.yml files?

Comment (VGalaxies) [reply .github/workflows/commons-ci.yml:1]: @imbajin which perfer?

Comment (imbajin) [reply .github/workflows/commons-ci.yml:1]: image

prefer simple one

Comment (imbajin) [reply .github/workflows/commons-ci.yml:1]: > I prefer to use the component names directly to express the ci-job meaning precisely, so that newcomers who don't know the background can understand it quickly.

We could ask some newcomers for their opinions:
@haohao0103 @zhenyuT @ChenZiHong-Gavin @vichayturen @returnToInnocence @jasinliu @ZeeJJ123 @AnkiPu @HJ-Young @Thespica

Which CI name do u prefer to choose: ( `Graph-XX-CI` or `HugeGraoh-XX-CI`
image, u could choose to support it with emoji 🚀 

Or choose to unify to ↓:
```markdown
- HugeGraph-PD & HugeGraph-Store & HugeGraph-Hstore CI / pd (pull_request)
- HugeGraph-PD & HugeGraph-Store & HugeGraph-Hstore CI / store (pull_request)
- HugeGraph-PD & HugeGraph-Store & HugeGraph-Hstore CI / hstore (pull_request)
- HugeGraph-Server CI / build-server (rocksdb, 11) (pull_request)
- HugeGraph-Server CI / build-server (mysql, 11) (pull_request)
- HugeGraph-Server CI / build-server (hbase, 11) (pull_request)
- ....
- HugeGraph-Server CI / xxx-...
```

u could choose to support it with emoji  👀

Comment (VGalaxies) [reply .github/workflows/commons-ci.yml:1]: For the second option, I feel the CI name is too long. I think a compromise would be to change Graph to HugeGraph while keeping the rest of the format unchanged.

**Comment by VGalaxies** [comment]
After changing the asf file from build (memory, 11) to build-server (memory, 11), if the GitHub workflow file is also modified in this PR, it will prevent build (memory, 11) from running (required). Therefore, only the asf file is modified in this PR, with a follow-up PR to change the GitHub workflow file from build to build-server (also disable merge button). cc @imbajin

File: .github/workflows/commons-ci.yml
Comment (VGalaxies) [reply .github/workflows/commons-ci.yml:44]: TODO: reset use stage to false later

File: .github/workflows/commons-ci.yml
Comment (VGalaxies) [reply .github/workflows/commons-ci.yml]: Choose full format based on voting.

**Review by imbajin — APPROVED** [review]

**Comment by imbajin** [comment]
Remember this PR must use **`Merge`** (can't use **rebase** option)

**Review by coderzc — APPROVED** [review]

**Pull request merged**
